Content/calendar.css

```diff
@@ -0,0 +1,18 @@
+﻿#tableId
+{
+    width: 100%;
+}
+
+#tableId td, #tableId th {
+    border: 1px solid #ddd;
+    padding: 8px;
+}
+
+#tableId th {
+    padding-top: 12px;
```

Comment:
Use the shorthand notation whenever you are writing the properties, wherever possible:
`padding: 12px 0 12px 0;`